src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+function ChatLog({ entries, updateLikes }) {
+  const ChatEntryComponenet = entries.map((entry) => {
```

Comment:
This should start with a lower case. We only have our component definitions start with upper case.